Bottom: d75fd0c21fb721a502f77106f1f88a04626513b7
Top:    e0288eaa86bd1f8b35b925a1f54698497e586aeb
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-16 22:43:40 +1300

Refresh of stream-summary

---

diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index ce06b6f..147588d 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -692,9 +692,9 @@ class TestStreamSummary(TestCase):
 
     def test_inprogress_tests_generate_failures(self):
         result = StreamSummary()
-        # complete status codes.
+        # terminal successful codes.
         result.startTestRun()
-        result.status("foo", "finish")
+        result.status("foo", "success")
         result.status("bar", "skip")
         result.status("baz", "exists")
         result.stopTestRun()
